requirements.yml

```diff
@@ -108,11 +108,15 @@
   numpy_compiled_extensions: false
   python: '>2.6,<3'
 
-# Chokes on no source
-# - name: gwcs
-#   version: 0.1.dev44
-#   astropy_helpers: true
-#   numpy_compiled_extensions: true
+- name: gwcs
+  version: 0.5
+  astropy_helpers: true
+  numpy_compiled_extensions: false
+
+- name: asdf
+  version: 1.0.0
+  astropy_helpers: true
+  numpy_compiled_extensions: yes
```

Comment:
`numpy_compiled_extensions` should be `false` for `asdf` because it is a pure python package.